translations/localify/ui.json

```diff
@@ -1997,5 +1998,158 @@
+    "8月": "Aug",
+    "9月": "Sep",
+    "10月": "Oct",
+    "11月": "Nov",
+    "12月": "Dec",
+    "育成データ削除の確認": "Confirm training data deletion",
+    "育成データ削除を確認したため\nタイトルに戻ります": "Due to confirmation of training data deletion \nreturning to title",
+    "{0}億人": "{0}00,000,000 Fans",
+    "{0}万人": "{0}0,000 Fans",
+    "+{0}人": "+{0} Fans",
+    "+{0}万人": "+{0}0,000 Fans",
+    "おすすめトレーナーがいません": "No recommended trainers",
+    "1分以内": "In 1 min",
+    "ミニキャラ": "Chibi",
+    "おすすめトレーナーを探す": "Recommend trainers",
+    "ヒントを入手できるスキルはありません": "No skills to get hint",
+    "30分以内": "In 30 min",
+    "予想結果": "Predicted Result",
+    "ウマ娘のクラス": "Uma Musume Class",
+    "おまかせ編成しました": "Auto-composed",
+    "出走条件達成！": "Entry req met!",
+    "特にありません": "No special conditions",
+    "おまかせ選択しました": "Auto-selected",
+    "+{0} 人": "+{0} Fans",
+    "ログがありません": "No logs",
+    "レンタルを含める": "Include rentals",
+    "{0}〜{1}人": "{0}〜{1} Fans",
+    "このウマ娘で１度も育成を完了していません": "This Uma Musume has not completed any trainings",
+    "このシナリオでの育成を行っていません": "No trainings for this scenario",
+    "ストーリーマーク設定": "Bookmark Story",
+    "変更した内容でストーリーマークを設定します\nよろしいですか？": "Changing the specified story bookmarks. \nContinue?",
+    "マークするストーリーを選択してください": "Please select stories to bookmark",
+    "マーク設定": "Set Bookmarks",
+    "ねぇ～！ちょー強いカブトムシ、\n捕まえに行こうよ～！\nゴルシも持ってるんだって～！": "Hey～! Let's go catch a strong beetle～! \nGolshi has one too～!",
+    "ねぇ～！ちょー強いカブトムシ、捕まえに行こうよ～！　ゴルシも持ってるんだって～！": "Hey～! Let's go catch a strong beetle～! \nGolshi has one too～!",
+    "ちゃんと見ててね、トレーナー！\nぱぱーっと勝ってくるから、\n瞬きしたら見逃しちゃうかもよ～？": "Watch carefully, trainer! \nI'll win in a flash; if you blink, \nyou might miss it～",
+    "ちゃんと見ててね、トレーナー！　ぱぱーっと勝ってくるから、瞬きしたら見逃しちゃうかもよ～？": "Watch carefully, trainer! \nI'll win in a flash; if you blink, \nyou might miss it～",
+    "予 想": "Predict",
+    "トレーナー、お疲れ様っ。\nうむうむ、今日もボクのために\nご苦労であ～る♪": "Trainer, good job! \nHmmm, you worked hard for me \nagain today～♪",
+    "トレーナー、お疲れ様っ。うむうむ、今日もボクのためにご苦労であ～る♪": "Trainer, good job! \nHmmm, you worked hard for me \nagain today～♪",
+    "どうせボクが勝っちゃうし、\n今のうちに表彰台での台詞\n考えとこっかな～♪": "I'm going to win anyway, \nso maybe I'll think of a line to \nsay on the podium right now～♪",
+    "どうせボクが勝っちゃうし、今のうちに表彰台での台詞考えとこっかな～♪": "I'm going to win anyway, \nso maybe I'll think of a line to \nsay on the podium right now～♪",
+    "トレーニングの効果が20％上昇するレース中、基礎能力が4％上昇する": "20% increase in training effectiveness\n4% increase in basic ability during a race",
+    "トレーニングの効果が10％上昇するレース中、基礎能力が2％上昇する": "10% increase in training effectiveness\n2% increase in basic ability during a race",
+    "トレーニングの効果に増減なしレース中、基礎能力の増減なし": "No change in training effectiveness\nNo change in basic ability during a race",
+    "トレーニングの効果が10％減少するレース中、基礎能力が2％減少する": "10% reduction in training effectiveness\n2% reduction in basic ability during a race",
+    "トレーニングの効果が20％減少するレース中、基礎能力が4％減少する": "20% reduction in training effectiveness\n4% reduction in basic ability during a race",
```

Comment:
These are in the mdb `translations/mdb/scenario-item-desc-*` and can be removed

Reply:
Ahh, you're right. Can you change basic ability to stats then? That's what's used everywhere else.